scripts/build/zone-js-builder.js

```diff
@@ -11,6 +11,7 @@
 const {resolve} = require('path');
 const {chmod, cp, mkdir, rm, test} = require('shelljs');
 const {baseDir, bazelBin, bazelCmd, exec, scriptPath} = require('./package-builder');
+const zonePackageJson = require('../../packages/zone.js/package.json');
```

Comment:
Let's keep the imports alphabetically ordered:

```suggestion
const zonePackageJson = require('../../packages/zone.js/package.json');
const {baseDir, bazelBin, bazelCmd, exec, scriptPath} = require('./package-builder');
```

Reply:
Got it, just updated, thanks.